### PR TITLE
Add part of the test page for DNT-compliant trackers

### PR DESCRIPTION
**Pull request by fmarier**
https://bugzilla.mozilla.org/show_bug.cgi?id=1258039

Related to https://github.com/mozilla/itisatrap/pull/15

### CODE REVIEW

**Comment by fmarier** [comment]
@bensternthal r?

File: check-dnt.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+function toggleStatus(enabled) {
+    var status = window.document.getElementById("dnt-" + enabled);
+    var waiting = window.document.getElementById("dnt-waiting");
+    waiting.classList.add("hidden");
+    status.classList.remove("hidden");
+}
+
+if (navigator.doNotTrack == "yes") {
```
Comment (bensternthal) [inline check-dnt.js]: Testing locally i saw that "missing dnt signal" came up every time.

My browser returns an integer for navigator.doNotTrack

See https://developer.mozilla.org/en-US/docs/Web/API/Navigator/doNotTrack

Comment (fmarier) [inline check-dnt.js]: You're right, I'm testing this with Nightly and I see "1" now. I will check for both.

**Comment by fmarier** [comment]
@bensternthal This new version checks for both the old and the new style.

**Comment by bensternthal** [comment]
Weird I thought I R+ this...

**Pull request merged**